@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -50,7 +50,7 @@ export interface ClipPlanesEvaluator {
  * slightly different in each case. General algorithm sets near plane between camera and
  * ground level, while far plane is just calculated using scale and bias approach with far offset
  * and multiplier.
- * @deprecated Class contains the legacy (first and original) clip planes evaluation method, which
+ * @note Class contains the legacy (first and original) clip planes evaluation method, which
```

Comment:
Please keep the deprecated. If you really need to use this class, disable the depreacation warning in the place where you use it

Reply:
That is something I would like to discuss. The depreciation flag in InterpolatedClipPlanesEvaluator was added by me personally :), cause we agreed that it will be no longer used / needed, but it seems that we actually have the use case for it - atmosphere needs different near/far planes then entire globe / map. So either I will disable tslint here, or remove depreciation or implement something similar inside MapViewAthmosphere, but I guess if we had it why not use it...

Reply:
Okay, if you need it than remove the deprecation but please also adjust the comment (i.e. remove the `legacy (first and original)` and maybe also add that this is used internally for the atmosphere